CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.3.1 (2018-11-29)
+
+- Fixed select tags to display invalid states on touched [#38](https://github.com/blackbaud/skyux-theme/pull/38)
```

Comment:
What about... "Fixed native HTML select fields to use the appropriate CSS style when control value is invalid."

Reply:
Done